doc/architecture/architecture_overview.adoc

```diff
@@ -7,7 +7,8 @@ endif::[]
 OSI contains an object-based environment description that uses the message format of the https://github.com/protocolbuffers/protobuf/wiki[Protocol Buffer] library.
 Google developed and maintains the Protocol Buffer library.
 OSI defines top-level messages that are used to exchange data between separate models.
-Top-level messages define the `GroundTruth` interface, the `SensorData` interface, and – since OSI version 3.0.0 – the interfaces `SensorView`, `SensorViewConfiguration`, and `FeatureData`.
+Top-level messages define the `GroundTruth` interface, the `SensorData` interface, and – since OSI version 3.0.0 – the interfaces `SensorView` and `SensorViewConfiguration`.
+Further top-level messages that were added in later versions of OSI are `TrafficCommand`, `TrafficUpdate`, `MotionRequest`, and `StreamingUpdate`.
```

Comment:
@thomassedlmayer Don't we have TrafficCommandUpdate as well?

Reply:
The `TrafficCommandUpdate` is not mentioned anywhere in the documentation. It is even missing in Figure 3 on the architecture overview page. If it is supposed to be a top-level message, this should probably be included with additional documentation.

Reply:
TrafficParticipant: https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/sensor-model/spec/model_types.html